client/src/components/dashboard/Community/CertLinks.js

```diff
@@ -2,40 +2,40 @@ import React from 'react';
 import propTypes from 'prop-types';
 import { Icon } from './SocialLinks';
 
-const URL = "https://freeCodeCamp.com/";
+const FCC_URL = "https://freeCodeCamp.com/";
 
 const CertLinks = ({ handleClick, fccCerts, username }) => {
   return(
     <div className="right floated">
-        <span>
-          { fccCerts.Front_End &&
-            <a
-              target="_blank"
-              onClick={handleClick}
-              href={`${URL + username}/front-end-certification`}>
-              <Icon title="Frontend Certified" className="desktop small icon" />
-            </a> }
-          { fccCerts.Data_Visualization &&
-            <a
-              target="_blank"
-              onClick={handleClick}
-              href={`${URL + username}/data-visualization-certification`}>
-              <Icon title="Data Visualization Certified" className="bar chart small icon" />
-            </a> }
-          { fccCerts.Back_End &&
-            <a
-              target="_blank"
-              onClick={handleClick}
-              href={`${URL + username}/back-end-certification`}>
-              <Icon title="Backend Certified" className="database small icon" />
-            </a> }
-        </span>
+      <span>
+      { fccCerts.Front_End &&
+        <a
+          target="_blank"
+          onClick={handleClick}
+          href={`${FCC_URL + username}/front-end-certification`}>
+          <Icon title="Frontend Certified" className="desktop small icon" />
+        </a> }
+      { fccCerts.Data_Visualization &&
+        <a
+          target="_blank"
+          onClick={handleClick}
+          href={`${FCC_URL + username}/data-visualization-certification`}>
+          <Icon title="Data Visualization Certified" className="bar chart small icon" />
+        </a> }
+      { fccCerts.Back_End &&
+        <a
+          target="_blank"
+          onClick={handleClick}
+          href={`${FCC_URL + username}/back-end-certification`}>
+          <Icon title="Backend Certified" className="database small icon" />
+        </a> }
+      </span>
     </div>
   );
 }
 
 CertLinks.propTypes = {
-  fccCerts: (propTypes.object || propTypes.bool),
+  fccCerts: propTypes.object,
   username: propTypes.string.isRequired
```

Comment:
@bonham000 I changed this because for this to work you would need to use the `propTypes.oneOfType([...arr])` method, and pass in both propTypes. But also, this propTypes warning will not be an issue once we remove the fake users from the app. This warning is only being triggered because one of our mock users does not have their profile public anymore. In production, this user would not get passed our validations - however we may want to consider using the `oneOfType` and have it expect a boolean as well if we want to manually white list users like Quincy to let them use the app.